### DISCUSSION TITLE
Can't use Tailwind with CRA and typescript

### DISCUSSION THREAD

**Discussion by geniusit** [tailwindlabs/tailwindcss#6759] [Help] [answered]
Hi,

I ve just started a new project with CRA (5.0.0) : npx create-react-app my-app --template typescript

And followed this guide : https://tailwindcss.com/docs/guides/create-react-app

The CSS isn't rendered

Any help please

**Comment by NovusOpus** [comment, marked as answer]
Is your react-scripts also v5.0.0?

If so, can you share your package.json and a minimal reproduction of your project so we can troubleshot what's going on.
